packages/experimental/Shadow/src/Shadow.tsx

```diff
@@ -1,31 +1,25 @@
 import * as React from 'react';
 import { View } from 'react-native';
-import { ShadowProps } from './Shadow.types';
+import { ShadowProps, ShadowTokens, shadowName } from './Shadow.types';
+import { compressible, buildUseTokens, useFluentTheme } from '@fluentui-react-native/framework';
+import { shadowStyleFromTheme } from './shadowStyle';
 
-export const Shadow: React.FunctionComponent<ShadowProps> = (props: ShadowProps) => {
-  const noOffset = { width: 0, height: 0 };
-  const neutralShadowAmbient = { shadowOffset: noOffset, shadowOpacity: 0.12, shadowRadius: 2 };
-  const neutralShadowAmbientDarker = { shadowOffset: noOffset, shadowOpacity: 0.2, shadowRadius: 8 };
-  const ambientShadow = {
-    '2': neutralShadowAmbient,
-    '4': neutralShadowAmbient,
-    '8': neutralShadowAmbient,
-    '16': neutralShadowAmbient,
-    '28': neutralShadowAmbientDarker,
-    '64': neutralShadowAmbientDarker,
-  };
-  const keyShadow = {
-    '2': { shadowOffset: { width: 0, height: 1 }, shadowOpacity: 0.14, shadowRadius: 2 },
-    '4': { shadowOffset: { width: 0, height: 2 }, shadowOpacity: 0.14, shadowRadius: 4 },
-    '8': { shadowOffset: { width: 0, height: 4 }, shadowOpacity: 0.14, shadowRadius: 8 },
-    '16': { shadowOffset: { width: 0, height: 8 }, shadowOpacity: 0.14, shadowRadius: 16 },
-    '28': { shadowOffset: { width: 0, height: 14 }, shadowOpacity: 0.24, shadowRadius: 28 },
-    '64': { shadowOffset: { width: 0, height: 32 }, shadowOpacity: 0.24, shadowRadius: 64 },
+const useShadowTokens = buildUseTokens<ShadowTokens>(() => ({}), shadowName);
+
+export const Shadow = compressible<ShadowProps, ShadowTokens>((props: ShadowProps) => {
+  const { depth } = props;
+  const theme = useFluentTheme();
+
+  const shadowStyle = shadowStyleFromTheme(theme, 'shadow' + depth.toString());
+
+  return (extra: ShadowProps, children: React.ReactNode) => {
```

Comment:
yarn buildci is going to fail because I don't use 'extra' anywhere, but if I delete that parameter I get red dialog boxes. Honestly not sure what this is supposed to be for (copied from other examples of compressible) - I could just print it out or something to get the build to pass, but not sure if there's a better alternative (ex. is there a way to remove it entirely without getting errors, and add it back in later when I get to this part?)

Reply:
When I delete it I get this error:

<img width="603" alt="Screen Shot 2022-07-01 at 3 09 22 PM" src="https://user-images.githubusercontent.com/78454019/176973079-83b48177-ec43-4fca-8399-13f09427957f.png">

Putting an underscore works! Although I didn't need it with the merge props change. Will play around with it.